### PR TITLE
Do not return stat var group (dc/g/Root) root when get path

### PR DESCRIPTION
**Pull request by shifucun**
This is a dummy node for linking the top level stat var groups, but not used by client.

### CODE REVIEW

File: internal/server/statvar_group.go [hunk 1]
```diff
@@ -295,6 +296,9 @@ func (s *Server) GetStatVarPath(
 	for {
 		if parents, ok := s.cache.ParentSvg[curr]; ok {
 			curr = parents[0]
+			if curr == svgRoot {
```
Comment (pradh) [inline internal/server/statvar_group.go:299]: There will be a different PR for the current flow? (i.e., with full hierarchy showing up in the client)

**Review by pradh — APPROVED** [review]

File: internal/server/statvar_group.go (hunk 1 above)
Comment (shifucun) [reply internal/server/statvar_group.go:299]: I am thinking to do that in mixer. So instead of relying on triples cache, mixer will read from the full hierarchy, and return the queried node.
